docs/resources/azurerm_ad_user.md.erb

```diff
@@ -164,6 +168,37 @@ The principal name of the user.
 
 A string value that can be used to classify user types in your directory, such as 'Member' and 'Guest'.
 
+### Other Attributes
+
+There are additional attributes that may be accessed that we have not
+documented. Please take a look at the [Azure documentation](#-Azure-REST-API-version).
+Any attribute in the response may be accessed with the key names separated by
+dots (`.`). Given the example response in their documentation:
+
+```
+...
+  "preferredLanguage": "en-US",
+  "provisionedPlans": [
+    {
+      "capabilityStatus": "Enabled",
+      "provisioningStatus": "Success",
+      "service": "exchange"
+    },
+...
+```
+
+We may access `provisioningStatus` with:
+
+```
+its('provisionedPlants.first.provisioningStatus') { should eq "Success" }
+```
+
+The api may not always return keys if there is no data associated with that
+key. There may be cases where the deeply nested property may not have the
```

Comment:
For clarity, maybe the first sentence could be reworded more like "The API may not always return keys that do not have any associated data." (Also make sure API is all caps.)

Reply:
:point_up:

Reply:
Thanks I missed this one earlier. I do not plan to merge without addressing it.